_contentTemplates/common/get-started.md

```diff
@@ -13,59 +13,58 @@ The latest version of Telerik UI for Blazor is `{{site.uiForBlazorLatestVersion}
 #add-nuget-feed
 ### Add the Telerik NuGet Feed to Visual Studio
 
-Telerik UI for Blazor is distributed through our private NuGet feed.
```

Comment:
Effectively, it is. For tutorial-like articles it is best to make things clear, there are still many people who wan to reference individual `dll` files. For advanced users, we have other articles that tell them how they can download `nupkg` files on their own. I propose we keep this line in the tutorials.

Reply:
I see how important NuGet is as a distribution method. This brings again the question of having another article in the Getting Started section that covers the MSI scenario. Covering both in a single article makes the article too complex.

Reply:
We have the three installation methods (including msi and nuget) in three separate articles under Installation.

Reply:
Back to the beginning of the discussion - we cannot keep the original line because it introduces a conflict in this article (the article starts with instructions on how to download an installer and then claims that NuGet is the only distribution method). 
To underline the significance of NuGet, I can change the line to something like:
"The recommended distribution method for the Telerik UI for Blazor packages is Telerik's private NuGet feed."